_data/handles/jah.json

```diff
@@ -0,0 +1,6 @@
+
+{
+  "handle": "jah",
+  "image": "./avatars/axic.png",
```

Comment:
Oh, it seems this is the wrong avatar for the person 😅 

cc @ninek9 @itsmetechjay